introduction.md

```diff
@@ -5,3 +5,6 @@ This is my intruduction
 my favourite language is Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?